### DISCUSSION TITLE
ImageEntry/SpatieMediaLibraryImageEntry - How to get relationship image?

### DISCUSSION THREAD

**Discussion by francoism90** [filamentphp/filament#10108] [Help]
```
SpatieMediaLibraryImageEntry::make('model')
    ->collection('avatar')
```
The `model` relationship is a `morphTo`.

It doesn't seem to parse the relationship, because it's looking for a dot in the name.

This doesn't seem to be documented, hopefully someone can help us with the answer.

Thanks. :)

**Comment by untillnesss** [comment]
Hello, any updates ?

**Comment by untillnesss** [comment]
@danharrin
